lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -168,6 +180,12 @@ Repair query columns to run once per result
 
 has columns => (is => 'ro', isa => 'ArrayRef[Str]', required => 0);
 
+after 'columns' => sub {
```

Comment:
Same as with the selectable values:

1. do we really need the check?
2. and if we do: do we really need to do the check on every attribute access?